usagov/usagov-2021
# Usagov 1975 left nav current page is a link

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1975

## Description

This changes the leftnav's current item into a div instead of a link and adjusts the styles so that they apply to the div.

## Testing Instructions


- Visit any page with a left nav
- Confirm that the current item is a div instead of a link
- Confirm that the current page indicator is a visually hidden element
- The HTML of the current item should match this structure:
```
<div class="active-trail usa-current">
  <p class="usa-sr-only">current page</p>
  Branches of government
</div> 
```

## Type of Changes

  - Add screenshot showing what it should look like

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
